test/server/cards/06-CotE/GloriousVictory.spec.js

```diff
@@ -0,0 +1,131 @@
+                this.setupTest({
+                    phase: 'conflict',
+                    player1: {
+                        inPlay: ['shinjo-altansarnai', 'moto-juro'],
+                        hand: ['glorious-victory', 'captive-audience']
+                    },
+                    player2: {
+                        inPlay: ['adept-of-the-waves'],
+                        provinces: ['shameful-display', 'public-forum', 'endless-plains']
+                    }
+                });
+                this.shinjoAltansarnai = this.player1.findCardByName('shinjo-altansarnai');
+                this.motoJuro = this.player1.findCardByName('moto-juro');
+                this.gloriousVictory = this.player1.findCardByName('glorious-victory');
+                this.adeptOfTheWaves = this.player2.findCardByName('adept-of-the-waves', 'play area');
+                this.shamefulDisplay = this.player2.findCardByName('shameful-display', 'province 1');
+                this.publicForum = this.player2.findCardByName('public-forum');
+                this.endlessPlains = this.player2.findCardByName('endless-plains');
+                this.noMoreActions();
+            });
+
+            it('should trigger on breaking a province during a military conflict', function() {
+                this.initiateConflict({
+                    type: 'military',
+                    province: this.shamefulDisplay,
+                    attackers: [this.shinjoAltansarnai],
+                    defenders: [],
+                    jumpTo: 'afterConflict'
+                });
+                this.player1.clickPrompt('No');
+                expect(this.player1).toHavePrompt('Triggered Abilities');
+                expect(this.player1).toBeAbleToSelect(this.gloriousVictory);
+                expect(this.player1).toBeAbleToSelect(this.shinjoAltansarnai);
+                this.player1.clickCard(this.gloriousVictory);
+                expect(this.shinjoAltansarnai.isHonored).toBe(true);
+                expect(this.motoJuro.isHonored).toBe(true);
+                expect(this.adeptOfTheWaves.isHonored).toBe(false);
+            });
+
+            it('should not trigger on breaking a province during a political conflict', function() {
+                this.initiateConflict({
+                    type: 'political',
+                    province: this.shamefulDisplay,
+                    attackers: [this.shinjoAltansarnai],
+                    defenders: [],
+                    jumpTo: 'afterConflict'
+                });
+                this.player1.clickPrompt('No');
+                expect(this.player1).not.toHavePrompt('Triggered Abilities');
+            });
+
+            it('should trigger when Endless Plains fires', function() {
+                this.initiateConflict({
+                    type: 'military',
+                    province: this.endlessPlains,
+                    attackers: [this.shinjoAltansarnai],
+                    defenders: []
```

Comment:
`defenders: []` needs to be removed as `this.initiateConflict` will error as triggers are occuring before the declare defender step.